model_compression_toolkit/qunatizers_infrastructure/common/base_quantizer.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from typing import List
+from enum import Enum
+
+from model_compression_toolkit.core import common
+from model_compression_toolkit.core.common.quantization.node_quantization_config import NodeWeightsQuantizationConfig, \
+    NodeActivationQuantizationConfig, BaseNodeNodeQuantizationConfig
+from model_compression_toolkit.core.common.target_platform import QuantizationMethod
+
+
+class QuantizationPart(Enum):
+    Activation = 0
+    Weights = 1
+
+
+class BaseQuantizer:
+    def __init__(self,
+                 qunatization_config: BaseNodeNodeQuantizationConfig,
```

Comment:
Can we fix this class BaseNodeNodeQuantizationConfig name in the process? (should be BaseNodeQuantizationConfig)